core/src/consensus/tendermint/mod.rs

```diff
@@ -919,7 +919,6 @@ impl ConsensusEngine<CodeChainMachine> for Tendermint {
         {
             self.signer.write().set(ap, address, password);
         }
-        self.to_step(Step::Propose);
```

Comment:
If then, when it starts?

Reply:
The tendermint starts when the timer expires.
NetworkExtension for TendermintExtension's `on_initialize` set timer and the timer calls `to_step` function.